src/BuildCheck.UnitTests/EndToEndTests.cs

```diff
@@ -200,11 +201,51 @@ public void NoEnvironmentVariableProperty_Test(bool? customConfigEnabled, string
             customConfigData);
 
         string output = RunnerUtilities.ExecBootstrapedMSBuild(
-            $"{Path.GetFileName(projectFile.Path)} /m:1 -nr:False -restore -analyze", out bool success, false, _env.Output, timeoutMilliseconds: 120_000);
+            $"{Path.GetFileName(projectFile.Path)} /m:1 -nr:False -restore -analyze", out bool success, false, _env.Output);
```

Comment:
nit: Why to remove this timeout? Is some test running more that 2 minutes?

Reply:
The timeout is unnecessarily long. The default is 30 seconds and recently a high custom timeout started to be copy-paste spreaded over the code